src/SloganForm.js

```diff
@@ -1,24 +1,28 @@
-export default function SloganForm(props) {
-  // React forms are a pain! 
+import { useState } from 'react';
+
+  // This component takes in a prop called setSlogans, which is a function that takes in an array of slogans. 
+  // Also, this component also takes in a prop called slogans, which is an array of strings. //
+export default function SloganForm({ slogans, setSlogans }) {
```

Comment:
I think the problem might be that you are only passing `setSlogans` as a prop to this component, but are attempting to use both `slogans` and `setSlogans`.

Reply:
Thank you for helping me with this!